config/manager/kustomization.yaml

```diff
@@ -3,5 +3,7 @@ resources:
 
 images:
 - name: controller
-  newName: docker.io/k8ssandra/medusa-operator
-  newTag: latest
\ No newline at end of file
+  newName: docker.io/adejanovski/medusa-operator
```

Comment:
Perhaps from k8ssandra/medusa-operator in the merged version ?

Reply:
oops, indeed 😅

Reply:
Fixed ✅